pkg/test/e2e/utils/http.go

```diff
@@ -32,7 +32,7 @@ import (
 type Backoff wait.Backoff
 
 const (
-	apiRequestTimeout = 10 * time.Second
+	apiRequestTimeout = 30 * time.Second
```

Comment:
Please leave a comment on this, mentioning Azure.

Reply:
sure,

I can also revert to my first commit where I treat the azure test differently:
https://github.com/kubermatic/kubermatic/pull/10073/commits/180a6bae863dc3ca9a20aa063da942afdc7155f5

but I thought this is simpler. WDYT?